cli/js/40_jupyter.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+const core = globalThis.Deno.core;
+const internals = globalThis.__bootstrap.internals;
+
+import { denoNsUnstable } from "ext:runtime/90_deno_ns.js";
+
+function enableJupyter() {
+  denoNsUnstable.jupyter = {
+    async broadcast(msgType, content) {
+      await core.opAsync("op_jupyter_broadcast", msgType, content);
```

Comment:
Nit: `ensureFastOps` here.

Reply:
I think that makes no difference since it's using `serde_json::Value` which has no chance to ever be a fast call.

Reply:
It's better performing than `opAsync` in general because it doesn't use the spread operator -- we generate overloads for the correct number of args.

In this case it's not critical but generally we should avoid using opAsync.

Reply:
I'll change it in a follow up PR